### ISSUE TITLE
Opening a lease when there are no prices(lease/downpayment) provided by the Oracle - is allowed

### ISSUE THREAD

**Issue by desislavva** [nolus-protocol/nolus-money-market#69]
Missing error message when a borrower tries to open a lease without having a currency price (lease/downpayment) provided by the Oracle. Currently, the result of such an attempt is: **_tx resolved instead of rejected_** which causes the lease to be blocked in the  **_opening:buy_asset_** state. 

Related to: #50

**Comment by gmanev7** [comment]
@desislavva is this still reproducible? If not please close

**Comment by gmanev7** [comment]
> @desislavva is this still reproducible? If not please close

It is still there.
More details: there is a price for the downpayment currency but no price for the lease currency

**Comment by desislavva** [comment]
Still there. 

And one more thing. 

Due to the restrictions, we have introduced (min_asset, min_transaction), in case there are no prices when trying to open a lease, and in particular, there is no price for the downpayment currency - the following error occurs: _"The asset amount should worth at least 15000 USDC_AXELAR:"._ 

This error is not a sufficient description of the problem. Personally, I tried to increase the amount I was sending until I noticed that there were no prices.

**Comment by gmanev7** [comment]
indeed, it's a good UX to prevent the lease from getting stuck on the buy asset step ... but even if we add such a validation we would not eliminate the probability of no prices at the time the lease opening gets to that step
